jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -5,6 +5,7 @@ def run(params) {
         env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
         // The junit plugin doesn't affect full paths
         junit_resultdir = "results/${BUILD_NUMBER}/results_junit"
+        env.exports = "export BUILD_NUMBER=${BUILD_NUMBER}; export BUILD_VALIDATION=true;"
```

Comment:
I like it.
Not for this PR maybe, but can you please align other pipelines to use a similar variable for the exports?